helmfile/charts/grafana-ops/values.yaml

```diff
@@ -26,3 +26,32 @@ dashboards:
     enabled: true
   networkpolicy:
     enabled: true
+  daily:
+    enabled: true
+
+daily:
+  limits:
+    influxdb:
+      workloadClusterKB: 4500000
+      serviceClusterKB: 4500000
+  thresholds:
+    # Thresholds for the number of elasticsearch snapshots
+    elasticsearchSnapshots:
+      critical: 20
+      warning: 16
```

Comment:
Right, this is a good point. I was thinking that too few would be detected in the "last modified object" pane, but not sure. Do you think it will be enough? I mean if there are no snapshots made then the last modified object will just get older and older and trigger its own threshold.

Reply:
Together with "last modified object" I think it is probably enough to look at "too many" here.